actions/copy/copy.js

```diff
@@ -40,7 +40,7 @@ async function main(args) {
         } else if (!adminPageUri) {
             payload = 'Required data is not available to proceed with FG Copy action.';
             logger.error(payload);
-            payload = await updateStatusToStateLib(projectPath, PROJECT_STATUS.FAILED, payload, undefined, COPY_ACTION);
+            payload = await updateStatusToStateLib(projectPath, PROJECT_STATUS.FAILED, payload, undefined, undefined, undefined, COPY_ACTION);
```

Comment:
Nit: Quite a bit of arguments for a function. May be we can simplify this. But for now it is fine.